inc/saihostif.h

```diff
@@ -370,6 +370,14 @@ typedef enum _sai_hostif_trap_type_t
      */
     SAI_HOSTIF_TRAP_TYPE_BFDV6 = 0x00004006,
 
+    /**
+     * @brief Intermediate System-to-Intermediate System (IS-IS) protocol
+     * traffic (L1 IS Device: 01:80:c2:00:00:14, L2 IS Device: 01:80:c2:00:00:15,
+     * All IS device: 09:00:2b:00:00:05, All ES device: 09:00:2b:00:00:04).
+     * Default packet action is trap
+     */
+    SAI_HOSTIF_TRAP_TYPE_ISIS = 0x00004007,
+
```

Comment:
ISIS is layer 2 AFAIK
And the trap is added here in the IP2ME section of traps
Please add in switch traps before SAI_HOSTIF_TRAP_TYPE_SWITCH_CUSTOM_RANGE_BASE
In addition, default action of trap, IMO, isn't backward compatible. Should be default forward